### PR TITLE
clear up the README that the master branch is using Vue2 bei default now

### PR DESCRIPTION
**Pull request by vuchl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ This is a project template for [vue-cli](https://github.com/vuejs/vue-cli). **It
 
 ``` bash
 $ npm install -g vue-cli
+# the master branch now is working with vue 2
```
Comment (LinusBorg) [inline README.md]: since this is already mentioned in Line 5 of the README, this should be removed.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,9 @@ This is a project template for [vue-cli](https://github.com/vuejs/vue-cli). **It
...
+$ vue init webpack my-project
+# for using vue ^1.0.0 && < 2.0.0
 $ vue init webpack#1.0 my-project
```
Comment (LinusBorg) [inline README.md]: Lines 20 and 21 can also be removed, this is already explained in Line 5 of README

Comment (vuchl) [inline README.md]: Added 2 commits. should be clean now

**Comment by LinusBorg** [comment]
Thanks @vuchl :)

**Pull request merged**